Assets/Scenes/SampleScene.unity

```diff
@@ -205,11 +205,18 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 5ea89fc4b9a29fe469c090277c774218, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  piles: []
```

Comment:
Shouldn't it be pills/pill? instead of pile/piles?

Reply:
Fixed